src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -749,17 +737,10 @@ internal void Import()
             // if paging is not specified then keep the old V2 behavior
             if (skip == 0 && first == ulong.MaxValue)
             {
-                ulong item = 0;
                 while (!_deserializer.Done())
                 {
                     object result = _deserializer.Deserialize();
-                    if (item++ < skip)
```

Comment:
this code is not necessary as ulong.MaxValue is a very large 64-bit number and this check is really only if it overflows